Source/WebCore/rendering/updating/RenderTreeUpdater.cpp

```diff
@@ -421,7 +421,7 @@ void RenderTreeUpdater::updateElementRenderer(Element& element, const Style::Ele
 
         // display:none cancels animations.
         auto teardownType = [&]() {
-            if (!elementUpdate.animationsAffectedDisplay && elementUpdate.style->display() == DisplayType::None)
+            if (!elementUpdate.style->animationsAffectedDisplay() && elementUpdate.style->display() == DisplayType::None)
```

Comment:
Only place I see reading animationsAffectedDisplay bit is here? Why is it needed for pseudo elements?